packages/twenty-server/src/engine/metadata-modules/field-metadata/services/field-metadata-related-records.service.ts

```diff
@@ -150,7 +178,7 @@ export class FieldMetadataRelatedRecordsService {
         'view',
       );
 
-    return await viewRepository.find({
+    return viewRepository.find({
```

Comment:
why do we not need await anymore here ?

Reply:
Because we're directly returning a promise so `await` is not needed as it doesn't change anything

Reply:
@ijreilly @magrinj we had issues in the past where people were updating the code from
```typescript
return viewRepository.find({
...
```
to
```typescript
const views = viewRepository.find({
... // doing some new stuff between fetching the views and returning it
return views;
```
forgetting to put back the await and breaking the code so I usually put it anyway. Not a strong requirement though, we don't have consensus yet but just a FYI.
The only advantage I see is that it delegates the decision to await to the caller but in practice we don't benefit from this that often